### DISCUSSION TITLE
How xDrip saved my life, after a surgery

### DISCUSSION THREAD

**Discussion by Sweetea420** [NightscoutFoundation/xDrip#2198] [General]
[My post](https://m.facebook.com/story.php?story_fbid=pfbid02J3RzFzs7xUKccUn1TXofcd6mLeoRfo28sjDKPbC3jY7UiqHyobTSYfMs8monrWENl&id=100013788041803&sfnsn=mo) 
I will paste my post here edited as of 30/6/2022

PASTE:
How i saved my life and survived a 100% deadly doctors mistake, with the help of xDrip application, versa2 watch and Glance watchface!!!
(Posted by me, live, on Fk type 1 diabetes , now copied here, edited, for sharing with others)
Hey. ☹☹☹☹
I had sinus surgery, i m in the hospital.(alone)
They did mistake with insulin.
They write 5u but its not, maybe 50.

After 6 hours of actrapid, i have extremely lows, double downs, with all actions taken.

(Edit: i had about 220, they gave me 5u intravenus actrapid!!!!!)

I had 2* dextrose bottles and 2 times dextrose intravenous 20ml 35% AND glucagon (my own glucagon kit) AND about 7 juice boxes. (Edit:and intravenus glucose, they wrote)

But we cant stop it.

It drops so fast 15-20mmol every 5 minutes.

I dont want to die because of the anaisthisiologist

I am very angry and sad but mostly AFRAID

Update:
I had to consume nearly 300gr carbs
Glucagon
2 times dextrose 20ml 35%
2 bottles of dextrose.

Now after all these i have dextrose dripping at maximum and i have 130 blood sugar.

And all that without active insulin, surgery was 8 hours ago

Edit: with dextrose dripping i have 170 now. But its going down.. lol
I will remove dextrose again to see what happens maybe in 1 hour

I dont want to stay tonight in the hospital after what happent. I d rather be with my dog. However i will take the decision more responsibly

Update: i did 2 correction units before 10minutes and stopped dextrose at 210, now i have single down (after 10mins) dropping 14mmol/5 minutes. In 45 minutes, the correction will kick in.. What a nightmare.

I know i have to consume carbs to avoid ketoacidosis but without insulin? How i am supposed to eat now, i cant. They fked up, they destroyed so many parameters of my otherwise good control. Just 1 person, the anaisthiologist, i told her :"are you fkn serious you try to kill me" and she was just sorry.

She is now at her home and her problems while i try to save my life constantly, instead of relaxing to heal from the surgery

More edit:
I am still going low.
I am leaving this hospital in 30 minutes.
I also had another plate of food, spaghetti, i didnt bolus and i am 60s(also while drinking the fifth juice)

End of this post:
I am home now at 20:30
I will be taking care of myself now.
I know to control unlike them.
Partially thnx to groups with ppl like you all.

Thank you. I ll be ok. I need rest now

Edit after 3 days: i am ok now, sugar wise.

Please be extra cautious with the anaisthesiologists. 
They can just end your life. It will say reason of death:"complications of t1 diabetes" 😥

Timeline:
00:00 Surgery ends
00:01 5u ACTRAPID intravenus by mistake, for 220 sugar!!
We go to 1st floor, from 3rd floor of surgery room.
00:10 nurse in front desk, receives me with 140 blood sugar and asleep, they put me on my bed, where xDrip gets connected!!!
00:20 i manage to wake up from anaisthisia because of alerts of my xDrip++ and my versa2 watch which was in the drawer, my blood sugar is 40 with double down arrows and my fight to save my life starts.

Surgeon said, at 00:40 you would be dead.
Also, without xDrip, or miaomiao, or versa2 or if they failed to connect this time, you would be dead.

I was left at the bed to wake up from anaisthisia unchecked.

I am very proud to be here today and in the name to all those who died of insulin OVERDOSE by doctors, i make my story public.

**Comment by Sweetea420** [comment]
How is it spam if they told me to post it here too from fb. Its my story

**Reply by Navid200** [reply]
There may be a misunderstanding.  
I never claimed that your post was a spam.  I did ask you to post here and thanks for that.

**Comment by Sweetea420** [comment]
Ow sorry i rushed to reply 😊
Happy new year
